### ISSUE TITLE
As an IO user, when I click the My Operator tile, I see the appropriate page depending on my status

### ISSUE THREAD

**Issue by BCerki** [bcgov/cas-registration#441]
#### Description:

Currently, whenever an industry user clicks on the "My Operator" tile, they're sent to the select-your-operator workflow. They should see a different message or start a different workflow depending on their status.

#### Acceptance Criteria:

Given I am an IO user
When my access to my operator is approved and I click the "My Operator" tile
Then I see a readonly version of the user operator form
And I see an Edit button
And when I click the edit button, I can edit the user operator form
(wireframe: https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=2414%3A1918&mode=design&t=xCpgH6PJIretcBQL-1)

Given I am an IO user
When my access to my operator is draft or rejected and I click the "My Operator" tile
Then I see the select-your-operator workflow (already done, just recording here for clarity and to ensure no bugs are introduced)

Given I am an IO user
When my access to my operator is pending and I click the "My Operator" tile
Then I see a pending message (covered in https://github.com/bcgov/cas-registration/pull/439)

#### Development Checklist:

- [x] Add conditional logic to `dashboard/select-operator` to show the user operator form for approved users
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Notes:

- Blocked by #285 
- Blocking #325

**Comment by hannavovk** [comment]
RE "Given I am an IO user When my access to my operator is approved and I click the "My Operator" tile Then I see the user operator form (**editable or readonly??????? wireframe shows an edit button**)":
- Editable if thy click the "Edit" button (in the future we might restrict editability for certain user types if we introduce a 'read-only' user type)
cc: @BCerki @marcellmueller @andrea-williams

**Comment by BCerki** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Team-Moose-Sprint-Board-60d216175a2689000ec774d0/issues/bcgov/cas-registration/441?planning-poker) with Zenhub @andrea-williams @BCerki @marcellmueller @JoshLarouche @Sepehr-Sobhani @shon-button

**Comment by Dianadec** [comment]
Related to [#338](https://app.zenhub.com/workspaces/team-moose-sprint-board-60d216175a2689000ec774d0/issues/gh/bcgov/cas-registration/338) possibly - as per refinement on Dec 21

Topic: "what does an approved operator see when they click the "my operator" tile:"

**Comment by Dianadec** [comment]
@hannavovk  fyi we discussed this ticket during standup, and Sepehr and Marcel will be refining it to capture the work that has been done, and will split out any outstanding work involved.
